src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -4390,10 +4390,13 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
 
                         - CODE GENERATION -
  /debug[+|-]                   Emit debugging information
- /debug:{full|pdbonly}         Specify debugging type ('full' is default, and 
+ /debug:{full|pdbonly|portable}
+                               Specify debugging type ('full' is default, and 
                                enables attaching a debugger to a running 
-                               program)
+                               program. 'portable' is a cross-platform format)
  /optimize[+|-]                Enable optimizations (Short form: /o)
+ /deterministic                Produce a deterministic assembly
```

Comment:
I'd say something "ensures that compiler outputs are determined entirely by inputs"

Otherwise it sounds like without this switch output code would have random behavior.